src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+}
+
+p,
+h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
   margin: 0;
+  padding: 0;
+
+  --white: #fff;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__info {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+  }
+
+  &__image {
+    margin: 0 auto 40px;
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+}
+
+.reviews {
+  min-width: 166px;
+  display: flex;
+  margin: 16px 0 24px;
+
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    display: flex;
+    justify-content: space-between;
+    padding-left: 17px;
+  }
+}
+
+.stars {
+  min-width: 60%;
+  line-height: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 60%;
```

Comment:
The width of the `.stars__star` is set to 60%, which might cause layout issues if the parent container's width is not appropriately set. Consider setting a fixed width or ensuring the parent container's width is defined to avoid unexpected behavior.